src/main/java/com/gregtechceu/gtceu/core/mixins/LevelRendererMixin.java

```diff
@@ -136,11 +154,29 @@ public static void renderShape(PoseStack poseStack, VertexConsumer consumer, Vox
                                                  double camX, double camY, double camZ, BlockPos pos,
                                                  BlockState state) {
         assert level != null;
+        var rendererCfg = ConfigHolder.INSTANCE.client.renderer;
+        int rgb = 0;
+        var condition = false;
```

Comment:
pls no `var primitive` ;-;
```suggestion
        boolean tintByColor = false;
```